src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -99,15 +99,29 @@ private void replaceText(String text) {
     private void handleCommandEntered() {
         try {
             commandExecutor.execute(commandTextField.getText());
-            initHistory();
-            historySnapshot.next();
-            commandTextField.setText("");
+            processCommandSuccess();
```

Comment:
Note to selves that this will be the second time this function is called. The first time is in the line above, within the ```MainWindow```'s ```executeCommand```.